worker-taskunstowing/src/main/java/com/github/jobservice/workers/taskunstowing/database/StowedTaskColumnName.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2021 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.jobservice.workers.taskunstowing.database;
+
+public final class StowedTaskColumnName
```

Comment:
Would this be better as an enumeration?

Reply:
I don't think it can be an enum as it's referenced in an annotation and so has to be a constant value:

https://github.com/JobService/worker-taskunstowing/blob/SCMOD-12730/worker-taskunstowing-container/src/test/java/com/github/jobservice/workers/taskunstowing/StowedTaskDAO.java#L27